tox.ini

```diff
@@ -23,11 +24,7 @@ commands = bash -c "\
                 {toxinidir}/"
 
 [testenv:flake8]
-deps =
-    flake8==3.5
-    pep8-naming==0.8.2
```

Comment:
where do you get the dependencies from?

Reply:
the default is little bit more up, by default installing everything from `requirements.txt`. It installs a little bit more than needed, but I think the advantages outweigh that one disadvantage:

 1) the extra stuff is going to be installed for another testenv anyway, so it is cached already,
 1) the dependencies are not listed in two different places,
 1) it makes the `tox.ini` file little bit more concise.